NewsApp/NewsViewControllers/Controllers/NewsViewController.swift

```diff
@@ -7,9 +7,28 @@
 
 import UIKit
 
-class NewsViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate {
-
+class NewsViewController: UIViewController {
+    
     let viewModel = NewsViewModel()
+    let categories: [NewsCategories] = [.business, .sports, .politics, .technology, .health, .science, .entertainment, .general]
+    let countries: [NewsCountry] = [.za, .us, .gb, .ca, .ch, .fr, .ru]
```

Comment:
You can move these here to the ViewModel since they are data that can be stored there.